src/js/components/AccordionPanel.js

```diff
@@ -0,0 +1,74 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from './Box';
+import Heading from './Heading';
+import ListItem from './ListItem';
+import OpenIcon from './icons/base/Add';
```

Comment:
I'd like to review the use of '+' and '-' as the right metaphors for the accordion controls with Chris. We use + and - for NumberInput where it means something entirely different. I'm wondering if up and down arrow heads would be better.